test/test-applications/azure-functions/Directory.Build.props

```
@@ -0,0 +1,48 @@
+<Project>
+  <PropertyGroup>
+    <!-- only run .NET Framework tests on Windows -->
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp3.1;net5.0</TargetFrameworks>
```

If the values are the same for all OSes, no need to have the conditions.

```suggestion
    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
```